src/components/Books.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+
+const Books = ({ category, title, author }) => (
```

Comment:
- [ ] [OPTIONAL] this component should ideally be name a `Book` since it is representing a the view of a single book

Reply:
Thanks noted